lib/plug/cowboy/handler.ex

```diff
@@ -4,6 +4,13 @@ defmodule Plug.Cowboy.Handler do
   @already_sent {:plug_conn, :sent}
 
   def init(req, {plug, opts}) do
+    start = System.monotonic_time()
+
+    :telemetry.execute([:plug_adapter, :request, :start], %{
+      name: :plug_cowboy,
```

Comment:
It looks like for this call, `name` is in the measurements, but for the other one, `name` is in the metadata. Shouldn't it always be in the metadata?

Reply:
Also, the `conn` that is created just below this has important information in it (like the path). I think we should probably include `conn` in the metadata for the start event